sources/packages/backend/apps/api/src/route-controllers/student/student.external.controller.ts

```diff
@@ -48,77 +41,35 @@ export class StudentExternalController extends BaseController {
   async searchStudentDetails(
     @Body() payload: StudentSearchAPIInDTO,
   ): Promise<StudentSearchResultAPIOutDTO> {
-    const studentPromise = this.studentInformationService.getStudentBySIN(
+    // Get student from SIMS.
+    const student = await this.studentInformationService.getStudentBySIN(
       payload.sin,
     );
+    // Get student applications from SIMS.
+    const studentApplicationsPromise =
+      student &&
+      this.studentInformationService.getStudentApplications(student.id);
     const sfasIndividualPromise =
       await this.studentInformationService.getSFASIndividualBySIN(payload.sin);
```

Comment:
Should the await be removed since it will be awaited later?

Reply:
that was a mistake 😊